spec/delayed/job_spec.rb

```diff
@@ -224,10 +226,12 @@ def create_job(opts = {})
         expect(query.to_sql).to match_sql(<<~SQL)
           SELECT "delayed_jobs".*
           FROM "delayed_jobs"
-          WHERE (((run_at <= '2025-11-10 17:20:13' AND (locked_at IS NULL OR locked_at < '2025-11-10 16:59:43')) OR locked_by = 'worker1')
-            AND failed_at IS NULL)
+          WHERE ("delayed_jobs"."run_at" <= '2025-11-10 17:20:13'
+              AND ("delayed_jobs"."locked_at" IS NULL OR "delayed_jobs"."locked_at" < '2025-11-10 16:59:43')
+              OR "delayed_jobs"."locked_by" = 'worker1')
```

Comment:
```
(((A <= v1 AND (B IS NULL OR B < v2)) OR C = v3) AND D IS NULL)
```

vs

```
(A <= v1 AND (B IS NULL OR B < v2) OR C = v3) AND D IS NULL
```

Could use someone to spot-check the parenthesis and operator precedence here, but I think these are logically identical 😅